### PR TITLE
[Chore] Prep for 0.6.20 maintenance release

### PR DESCRIPTION
**Pull request by zquestz**
Build with latest Go release and update deps.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 language: go
 go:
-  - 1.12
-  - tip
```
Comment (emergent-reasons) [inline .travis.yml:4]: Was it depending on tip for anything? I assume we are far past whatever it was if there was something anyway.

Comment (zquestz) [reply .travis.yml:4]: Nope, just was something to test the latest releases. =)

**Review by emergent-reasons — APPROVED** [review]
go has come a long way since the last release.

Tests pass with old and new go versions. LGTM

File: server/e2e_test.go [hunk 2]
```diff
@@ -549,15 +549,19 @@ func (h *testHarness) WaitBroadcastVerificationKey(vk string, pool []*testClient
...
 		actualPacket, err := c.inbox.PopOldest()
 		if err != nil {
 			h.t.Fatal(err)
 		}
 		assert.Len(h.t, actualPacket.message.GetPacket(), 1)
 		actualMsg := actualPacket.message.GetPacket()[0]
-		assert.Equal(h.t, expected, actualMsg)
+		// Can't compare the exact object, as there are protobuf related fields that should
+		// not be compared! Instead just make sure the accused key matches!
+		assert.Equal(h.t, expected.Packet.Message.Blame.Accused.Key, actualMsg.Packet.Message.Blame.Accused.Key)
```
Comment (emergent-reasons) [inline server/e2e_test.go:561]: So this test was just luckily passing before? Or new protobuf has additional fields?

Comment (zquestz) [reply server/e2e_test.go:561]: New fields that it uses internally. You can see them in the newly generated file. Everything should still work fine, we just had to adjust this one test expectation. =)

**Pull request merged**